CMakeLists.txt

```diff
@@ -85,12 +85,17 @@ set_target_properties(coordgen
         SOVERSION 1
 )
 
+if(MSVC)
```

Comment:
Why is this MSVC block here? If it's not present, are the CMAKE_INSTALL_LIBDIR and CMAKE_INSTALL_BINDIR unreliable?

Reply:
CMAKE_INSTALL_LIBDIR and CMAKE_INSTALL_BINDIR are set by [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html).  Presumably, this isn't set in MSVC since this PR was failing CI in MSVC without those variables being set.

Reply:
Thanks, Peter.